apps/authentication/mfa/face.py

```diff
@@ -30,10 +30,11 @@ def is_active(self):
 
     @staticmethod
     def global_enabled():
-        return settings.XPACK_LICENSE_IS_VALID \
-            and LicenseEditionChoices.ULTIMATE == \
-            LicenseEditionChoices.from_key(settings.XPACK_LICENSE_EDITION) \
-            and settings.FACE_RECOGNITION_ENABLED
+        return (
+            settings.XPACK_LICENSE_IS_VALID and
+            settings.XPACK_LICENSE_EDITION_ULTIMATE and
+            settings.FACE_RECOGNITION_ENABLED
+        )
 
     def get_enable_url(self) -> str:
         return '/ui/#/profile/index'
```

Comment:
In contrast to the revised code snippet that was checked on 2021-09-01, this new version introduces an additional condition within the `global_enabled` staticmethod:

```
def global_enabled():
    return (settings.XPACK_LICENSE_IS_VALID 
           and LicenseEditionChoices.ULTIMATE == 
              LicenseEditionChoices.from_key(settings.XPACK_LICENSE_EDITION)  
           and settings.FAC
```